ApollosProject/apollos-apps
# Adds CampaignItems Algorithm

## DESCRIPTION

<img src="https://user-images.githubusercontent.com/1637694/76346886-1c93d900-62dc-11ea-8a6b-1d7c8aaf061e.png" width="30%" />

### What does this PR do, or why is it needed?

Adds an algorithm for "Campaign Items", or as we call them on the home feed, "Featured Items". Right now the only way to render this algorithm is inside a `ActionListCard`, but that will be changing soon.

@burkeshartsis - I believe you wanted to see this PR for reference on how to implement an algorithm. 

### What design trade-offs/decisions were made?

### How do I test this PR?

1. Clone this PR.
2. Clone this branch in the packages repo
https://github.com/ApollosProject/apollos-templates/pull/4
3. Open the home feed.
  - you should see an action list card with identical content as the featured card.

### What automated tests did you write?

Tests over new datasource methods.

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
